metadata-models/src/main/pegasus/com/linkedin/ml/metadata/MLModelEndpointProperties.pdl

```diff
@@ -0,0 +1,43 @@
+namespace com.linkedin.ml.metadata
+
+import com.linkedin.common.Time
+import com.linkedin.common.VersionTag
+import com.linkedin.common.CustomProperties
+
+/**
+ * Properties associated with an ML Model Endpoint
+ */
+@Aspect = {
+  "name": "mlModelEndpointProperties"
+}
+record MLModelEndpointProperties includes CustomProperties {
+
+  /**
+   * Documentation of the MLModelEndpoint
+   */
+  @Searchable = {
+    "fieldType": "TEXT",
+    "hasValuesFieldName": "hasDescription"
+  }
+  description: optional string
+
+  /**
+   * Date when the MLModelEndpoint was developed
+   */
+  date: optional Time
+
+  /**
+   * Version of the MLModelEndpoint
+   */
+  version: optional VersionTag
+
+  /**
+   * Tags for the MLModelEndpoint
+   */
+  tags: array[string] = [ ]
```

Comment:
let's remove this- tags we apply via a GlobalTags aspect now. This is a legacy tagging format